datahub/main.py

```diff
@@ -123,3 +124,23 @@ def get_dsr_data(  # type: ignore[misc]
     filtered_data = dt.dsr_data[start : end + 1 if end else end]
 
     return {"data": filtered_data}
+
+
+@app.get("/wesim")
+def get_wesim_data() -> dict[Hashable, Any]:  # type: ignore[misc]
+    """GET method function for getting Wesim data as JSON.
+
+    Returns:
+        A Dict containing the Wesim Dataframes
+    """
+    if dt.wesim_data == {}:
+        dt.wesim_data = get_wesim()
+
+    data = {
+        "Capacity": dt.wesim_data["Capacity"],
+        "Regions": dt.wesim_data["Regions"],
+        "Interconnector Capacity": dt.wesim_data["Interconnector Capacity"],
+        "Interconnectors": dt.wesim_data["Interconnectors"],
+    }
+
+    return {"data": data}
```

Comment:
We don't need to get each element like this.
```suggestion
    return {"data": dt.wesim_data}
```